backend/main.go

```diff
@@ -72,11 +73,13 @@ func newSteveServer(c stevecli.Config) (*steveserver.Server, error) {
 		return nil, err
 	}
 	restConfig.RateLimiter = ratelimit.None
+	router := apirouter.Router{Kubeconfig: c.KubeConfig}
 	a := auth.NewK3sAuthenticator(restConfig.Host)
 	return &steveserver.Server{RestConfig: restConfig,
 		AuthMiddleware: auth.ToAuthMiddleware(a),
 		DashboardURL: func() string {
 			return c.DashboardURL
 		},
+		Router: router.Routes,
```

Comment:
can u please rebase with the master branch and using `Next Handler` of new routers https://github.com/guangbochen/edge-api-controller/blob/master/backend/main.go#L85